README.md

```diff
@@ -9,97 +9,117 @@ Issues on Github will be disabled soon.**
 
 # Contributing
 
-- [Styling and formatting your contribution](#styling)
+If this is your first contribution to an open source software (OSS) project, congratulations! Contributing to the docs is a great way to get started; you can learn more about DC/OS, and contribute back to an OSS project right away. No expertise necessary. If you are an old hand, welcome back!
+
+Below you will find some instructions on how to contribute to the DC/OS docs. By following them you help make sure that no one unnecessarily duplicates work, and that the people who can merge (approve and integrate) your contribution are aware that you’re working on something. In order to follow these instructions you’ll need to have a [GitHub account](https://github.com/join?source=header-home) and an account on the [DC/OS JIRA](https://dcosjira.atlassian.net/admin/users/sign-up). If you aren’t signed up for those things yet go do it now. We’ll be here when you get back.
+
 - [Making your contribution](#making)
+- [Styling and formatting your contribution](#styling)
 - [Building and testing your content locally](#test-local)
-- [Submitting your pull request](#submitting)
 
+## <a name="making"></a>Making your contribution
 
-## <a name="styling"></a>Styling and formatting your contribution
+1. Search [JIRA](https://dcosjira.atlassian.net/browse/DCOS-508?jql=project%20%3D%20DCOS%20AND%20component%20%3D%20documentation) to make sure that no one is already working on your issue. If you find your issue and it isn’t assigned to anyone, it’s free: assign it to yourself! If you don’t see the issue you want to work on, [create a new issue](https://dcosjira.atlassian.net/secure/CreateIssue!default.jspa), select documentation as the component, and assign it to yourself. That way other contributors will know you are working on it.
 
```

Comment:
no, linking to the URL wouldn't render. to add a graphic you'll need to create an images directory, then link to it (e.g. https://github.com/dcos/dcos-website/tree/develop/img).